roles/jd-client/src/job_declarator/mod.rs

```diff
@@ -201,15 +203,22 @@ impl JobDeclarator {
         self_mutex: &Arc<Mutex<Self>>,
         template: NewTemplate<'static>,
         token: Vec<u8>,
-        tx_list: Seq064K<'static, B016M<'static>>,
+        tx_list_sequence: Seq064K<'static, B016M<'static>>,
```

Comment:
no need to have list and sequence in the name chose one and stick with it: `tx_list` or `tx_sequence`.

Personally I prefer `tx_list`.